packages/apollo/src/commands/service/download.ts

```diff
@@ -1,12 +1,12 @@
 import { flags } from "@oclif/command";
 import { introspectionFromSchema } from "graphql";
 import { writeFileSync } from "fs";
-
 import { ProjectCommand } from "../../Command";
 
 export default class ServiceDownload extends ProjectCommand {
   static aliases = ["schema:download"];
-  static description = "Download the schema from your GraphQL endpoint.";
+  static description =
+    "Download a schema from a locally running GraphQL endpoint.";
```

Comment:
I think this is safe to leave as is - you can download from any endpoint. You just can't use this command to download from engine.

Reply:
You _can_ but I added that just to clarify the _purpose_ of the command.

Reply:
Do we consider it an improper use case to download the schema from a graph that isn't running locally? I actually don't know the answer to that question, but I'd imagine there's a valid use case in there.

I think downloading from a locally running server is just one use case, but really this command is set up to handle others as well.